content/tutorials/manage-the-rippled-server/installation/build-run-rippled-macos.md

```diff
@@ -24,7 +24,7 @@ For development purposes Ripple recommends running `rippled` as your own user, n
 
         $ brew install git cmake pkg-config protobuf openssl ninja
 
-0. Install Boost 1.67.0. `rippled` 1.2.0 is compatible with Boost 1.67. <!--#{ no boost@1.67 formula, so must manually compile and install. will eventually upgrade to boost 1.68, but needs testing first }# -->
+0. Install Boost 1.67.0 or higher. `rippled` 1.3.0 is compatible with Boost 1.67.0 and higher. The latest version of Boost available in the Homebrew repositories isn't new enough, so you must install Boost manually:
```

Comment:
Is it compatible with Boost 1.67.0? I got this error:
```
...
-- rippled version: 1.3.0
-- NIH-EP cache path: /Users/user/ripple/rippled/.nih_c/unix_makefiles/AppleClang_10.0.0.10001145/Debug
-- using [caa5c9e2232ff1eee1d0c61907283649e48e6fbc] as build container tag...
-- BOOST_ROOT: /Users/user/Downloads/boost_1_67_0
-- BOOST_LIBRARYDIR: 
CMake Error at Builds/CMake/FindBoost.cmake:2153 (message):
  Unable to find the requested Boost libraries.

  Boost version: 1.67.0

  Boost include path: /Users/user/Downloads/boost_1_67_0

  Detected version of Boost is too old.  Requested version was 1.70 (or
  newer).
Call Stack (most recent call first):
  CMakeLists.txt:717 (find_package)


CMake Error at CMakeLists.txt:1114 (get_target_property):
  get_target_property() called with non-existent target "Boost::date_time".


-- Pausing to download NuDB...
-- Could NOT find Doxygen (missing: DOXYGEN_EXECUTABLE) 
-- doxygen executable not found -- skipping docs target
-- Configuring incomplete, errors occurred!
...
```

I'm building Boost 1.70.0 now, to give it a try. Using:
```
./b2 cxxflags="-std=c++14" visibility=global
```
(The `visibility=global` is new.)